terraform/eval_variable.go

```diff
@@ -25,7 +26,7 @@ func evalVariableValidations(addr addrs.AbsInputVariableInstance, config *config
 	}
 
 	// Variable nodes evaluate in the parent module to where they were declared
-	// because the value expression (n.Expr, if set) comes from the calling
+	// because the value expression (expr, if set) comes from the calling
```

Comment:
`n.Expr` appears to have been changed to `expr` a while back so this is just fixing the docstring since I noticed it when fixing the merge conflicts from the original PR